#this tells the compiler to use gcc, gfort and mpif90/mpicc
diff --git a/Makefile b/Makefile
index 5fbaeca3..0aae0807 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-MODEL_FORMULATION = 
+MODEL_FORMULATION = gfortran 
 
 
 dummy:
